### ISSUE TITLE
Put mouse wheel sensitivity in Preferences

### ISSUE THREAD

**Issue by dfannius** [yishn/Sabaki#250]
The old mouse wheel sensitivity (before https://github.com/yishn/Sabaki/commit/6e0f5e336c39196b61355129f3fef7776eb622cd) was perfect for me (one "click" on my mouse wheel = one move). I am glad that I can return to the old behavior by hand by changing settings.json, but it would be nice for other users to make it easy to modify without having to dig into the debugging documentation.

**Comment by yishn** [comment]
I don't think that providing a way to change this setting would benefit a lot of users. The mouse wheel sensitivity is already an OS setting. At least in Windows, in 'Mouse Properties' > 'Vertical Scrolling', you can set the number of lines which is scrolled by one mouse wheel click. 

I have `2` as the value  there and it works fine here, i.e. one mouse wheel click = one move. If I change that to `1` I have to do two mouse wheel clicks for one move. I can reduce the value of `game.navigation_sensitivity` so that one only needs one mouse wheel click for one move if number of lines is set to `1`.

**Comment by dfannius** [comment]
My mouse wheel sensitivity is already at the setting I want it in all my other applications (in OS X), so I don't want to make a global change to my entire machine. But, as I said, there is already a way to do what I need in settings.json, so maybe I shouldn't guess at what other users want until they actually complain.

**Comment by wctgit** [comment]
I haven't been affected (yet) by this setting, but I would agree with @dfannius that it would be a nice enhancement to provide a GUI interface to change the setting for the behaviour of the mousewheel, independent of the OS setting. The OS setting is naively concerned with lines of text usually, but in many other applications, e.g. games, image editors, etc., the mouse wheel is handled independently, as I agree it should be handled independently by Sabaki. And the default should be one wheel 'click', one move, regardless of OS settings. This is, after all, part of Sabaki's default user interface, and one of Sabaki's primary unique features is its pleasant and aesthetic user interface, where things seem to work very 'naturally', as one would 'expect' them to, with few 'surprises'.

**Comment by yishn** [comment]
@wctgit In a perfect world, the mouse wheel sensitivity stuff should only affect continuous mouse wheels such as those on Apple's mice (e.g. a mouse where the wheel doesn't 'click'). If you haven't been affected, that's great.

The problem is, there's no way to determine whether you mouse has a continuous mouse wheel or not, the only accessible value in JS is a number (see [deltaY](https://developer.mozilla.org/en-US/docs/Web/API/WheelEvent/deltaY)). On my end, the absolute value of `deltaY` is `100` each 'click', so it always triggers a forward/backward navigation.

**Comment by wctgit** [comment]
I understand your points, and that this is a 'nice to have' enhancement idea, rather than high priority, but just to make sure I have a clear understanding about what's possible: Doesn't the existence of deltaY and your existing threshold of 40 mean that in the future there **could** be a part of the settings UI which allows the user to tweak some value (I guess the 'threshold') for their particular machine, to solve the problem in the OP? Or is this information still not enough to be 'fixed' in all cases? Just curious. :-)
